src/web/app/components/student-list/student-list.component.ts

```diff
@@ -66,6 +68,17 @@ export class StudentListComponent implements OnInit {
   ngOnInit(): void {
   }
 
+  ngAfterContentChecked(): void {
+    this.sortedStudents = [...this.students];
+    if (this.tableSortBy === SortBy.NONE) {
+      this.sortedStudents.sort(this.sortBy(SortBy.STUDENT_NAME))
```

Comment:
I don't think this is the perfect way to do chain sorting (as the order is actually reverse order). @madanalogy I remember you have worked on the chain sorting before?

Reply:
Yeap in #10235, you can chain them with the `||` operator, e.g:
https://github.com/TEAMMATES/teammates/blob/f4c7e6d1bf248ebf18d44214087ac28da300b891/src/web/app/components/question-responses/per-question-view-responses/per-question-view-responses.component.ts#L117-L123